AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/MutationSync/OutgoingMutationQueue/SyncMutationToCloudOperation.swift

```diff
@@ -109,7 +112,7 @@ class SyncMutationToCloudOperation: AsynchronousOperation {
         authType: AWSAuthorizationType? = nil
     ) -> GraphQLRequest<MutationSync<AnyModel>>? {
         var request: GraphQLRequest<MutationSync<AnyModel>>
-
+        let version = getLatestSyncMetadata()?.version
```

Comment:
let's keep the code aligned, however small.. from https://github.com/aws-amplify/amplify-swift/pull/3377/files#diff-e22aa4a69639447ae34d5937d1db5813f9d6c878e426206ef3bb85e4b0600973 would be 
```
let latestSyncMetadata = getLatestSyncMetadata()
```

Reply:
sure, will update the PR.